activity/views.py

```diff
@@ -440,6 +443,42 @@ def admin_user_management(request):
     )
 
 
+def admin_organization(request):
```

Comment:
Just move this logic to organization profile tab (There we will have organization details in an editable fields)